src/test/Components/CreateImageWizard/CreateImageWizard.content.test.tsx

```diff
@@ -158,10 +158,12 @@ describe('Step Packages', () => {
     await clickNext();
   };
 
-  test('clicking Next loads Image name', async () => {
+  test('pclicking Next loads Image name', async () => {
     await setUp();
 
-    await clickNext();
+    // skip packages
+    //  await clickNext();
+    // skip repositories
```

Comment:
if we don't need it, lets remove it, tho I do not understand why we have less steps now :thinking:

Reply:
actually I didnt understand too, and I saw that First Boot step didnt exist when I check it locally, is it possible?

Reply:
Some of it could be flags, but also did anyone reeeally count the steps before? If we got to Details step, clicked on disabled Next button three times and then filled in a name and continued with the button now enabled, the tests would pass, right? :sweat_smile: